python/rpdk/java/templates/init/guided_aws/TagHelper.java

```diff
@@ -0,0 +1,256 @@
+package {{ package_name }};
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Sets;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang3.ObjectUtils;
+import software.amazon.awssdk.awscore.AwsResponse;
+import software.amazon.awssdk.core.SdkClient;
+// TODO: Replace the Tag model below according to your resource
+import software.amazon.awssdk.services.cloudformation.model.Tag;
```

Comment:
What is this about?  Is it about the service SDK Tag model?

Reply:
I think it's pretty hard to have people to know this is the place to change.

Reply:
yeah we need them to use their own service Tag model, let me add more details in TODO maybe